project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.11
+sbt.version=1.0.0-RC3
```

Comment:
To unblock you I suggest you revert this change and cross-build from 0.13.16.

The scripted-plugin issue you're seeing has been highlighted by @smarter before, but hasn't been addressed yet.